LastPush/CanvasGapsGenerator/src/main/kotlin/jetbrains/kotlin/course/last/push/PreDefinedSymbols.kt

```diff
@@ -3,7 +3,7 @@
 package jetbrains.kotlin.course.last.push
 
 val separator = ' '
-val newLineSymbol = System.lineSeparator()
+val newLineSymbol = System.lineSeparator().replace("\r\n", "\n")
```

Comment:
It is a bad way to fix that - since if we remove this complex for the students part about .dropLastWhile { it.toString() == newLineSymbol } it will not work I suppose